spec.html

```diff
@@ -324,7 +322,7 @@
       <h1>BigInt::signedRightShift (_x_, _y_)</h1>
       <p>The abstract operation BigInt::signedRightShift with arguments _x_ and _y_ of type BigInt:</p>
       <emu-alg>
-        1. Return ? BigInt::leftShift(_x_, -_y_).
+        1. Return BigInt::leftShift(_x_, -_y_).
```

Comment:
Fine with me, but then the same should be done in the other definitions in 1.1 (that's 5 or 6 changes).